tox.ini

```diff
@@ -1,15 +1,15 @@
 # Copyright (C) 2019, Nokia
 
 [tox]
-envlist = py27, py36, docs, pylint
+envlist = py27, py37, docs, pylint
 
 [base]
 deps =
     mock
     pytest
     pytest-cov
     pytest-pep8
-    pytest-console-scripts
+    pytest-console-scripts==0.1.6
```

Comment:
Reason for changes to dependencies (and also pytest options) should be explained in commit message.
This (and pytest option changes) could be in same commit where pylint checks are disabled.

Reply:
ok